sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWriteUnshardedRecords.java

```diff
@@ -342,11 +348,13 @@ void flush(RetryManager<AppendRowsResponse, Context<AppendRowsResponse>> retryMa
                   this.currentOffset += inserts.getSerializedRowsCount();
                 }
                 ApiFuture<AppendRowsResponse> response = writeStream.appendRows(offset, protoRows);
-                inflightWaitSecondsDistribution.update(writeStream.getInflightWaitSeconds());
-                if (writeStream.getInflightWaitSeconds() > 5) {
-                  LOG.warn(
-                      "Storage Api write delay more than {} seconds.",
-                      writeStream.getInflightWaitSeconds());
+                if (!usingMultiplexing) {
```

Comment:
We are fixing this in this PR:
https://github.com/googleapis/java-bigquerystorage/pull/1835